### PR TITLE
Add references to Windows containers

### PR DESCRIPTION
**Pull request by RobDolinMS**
Signed-off-by: Rob Dolin <robdolin@microsoft.com>

### CODE REVIEW

File: config-windows.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Windows Application Container Configuration
+
+Windows application containers can be configured using the following properties.
```
Comment (wking) [inline config-windows.md]: Where's the meat?  Is this supposed to have stuff from #504?

Comment (RobDolinMS) [inline config-windows.md]: @wking To your question of ["Where's the beef"](https://en.wikipedia.org/wiki/Where%27s_the_beef%3F), there's not much here yet.  I would expect that #504 and similar would be added over time.  My desire with this PR it clear that it is the intent of OCI to support Linux, Solaris, and Windows.

Comment (wking) [inline config-windows.md]: On Thu, Sep 01, 2016 at 04:11:39PM -0700, Rob Dolin (MSFT) wrote:
> > @@ -0,0 +1,3 @@
> > +# Windows Application Container Configuration
> > +
> > +Windows application containers can be configured using the following properties.
>
> … My desire with this PR it clear that it is the intent of OCI to
> support Linux, Solaris, and Windows.

Then how about dropping this file (until you have Windows-specific
settings to put into it) and using:

  * Windows containers: [runtime.md](runtime.md) and [config.md](config.md)

in README.md?

**Comment by RobDolinMS** [comment]
Signed-off-by: Rob Dolin <robdolin@microsoft.com>

**Comment by RobDolinMS** [comment]
@opencontainers/runtime-spec-maintainers Per conversation on OCI Dev ConCall, please LGTM and merge.  Thanks.  :)

**Comment by wking** [comment]
On Wed, Sep 07, 2016 at 02:19:30PM -0700, Rob Dolin (MSFT) wrote:
> Signed-off-by: Rob Dolin <robdolin@microsoft.com>

Ah, looks like the Signed-off-by in d4076604a is missing a trailing
‘>’.  Can you amend and force-push?

**Comment by RobDolinMS** [comment]
Doh.  Thanks @wking.  I see my typo at the top of: https://github.com/opencontainers/runtime-spec/commit/d4076604a0d1050628b5b283337d1c7e37c756cc

With the tool chain I'm using, I can't easily amend and force-push.  :(

If two maintainer LGTMs cannot merge this change, please LMK and I'll re-create the PR and re-submit.

**Comment by tianon** [comment]
@RobDolinMS yeah, I think the DCO check is a firm blocker (AIUI, IANAL); if your current toolchain can't amend/force, could you clone your branch elsewhere and amend/force there?  Otherwise I think you're stuck with a resubmit. :disappointed:

**Comment by wking** [comment]
On Thu, Sep 08, 2016 at 01:50:48PM -0700, Tianon Gravi wrote:
> @RobDolinMS yeah, I think the DCO check is a firm blocker (AIUI,
> IANAL); if your current toolchain can't amend/force, could you clone
> your branch elsewhere and amend/force there?  Otherwise I think
> you're stuck with a resubmit. :disappointed:

I believe this has been dealt with in the past by having the merging
maintainer insert Rob's Signed-off-by in the commit message when
merging (in cases where he has provided that Signed-off-by via a
GitHub comment).  But I haven't dug up a specific instance, so I may
be miss-remembering.

**Comment by RobDolinMS** [comment]
Working on a replacement for this with more meat, and meeting with Windows Containers experts later today

**Comment by RobDolinMS** [comment]
Closing per above comments.

**Comment by wking** [comment]
On Wed, Sep 14, 2016 at 04:16:45PM -0700, Rob Dolin (MSFT) wrote:
> Closing per above comments.  

With the Windows changes coming in in a stream of tiny PRs (which I
like for focused review ;), somebody is still going to have to land
the README protocol reference you have here [1].  I suggest re-opening
this PR to land just that line, and have whoever lands the first meat
in a config-windows.md take care of the rest of what you have here
now.

[1]: https://github.com/opencontainers/runtime-spec/pull/554/files#diff-04c6e90faac2675aa89e2176d2eec7d8R35

**Pull request closed without merging**